### ISSUE TITLE
Load time performance

### ISSUE THREAD

**Issue by jread-usgs** [USGS-VIZLAB/water-use-15#62]
Putting this in for reference with our first dev deployment that was made off of [this commit](https://github.com/USGS-VIZLAB/water-use-15/commit/71998dc010e1b9a584860d6bc9654460ab840372)

From dev on a fresh browser (it caches, so the second time is faster), we have a 4.38s load time, and we don't see the map until that is complete. Seems to be fairly consistent

![image](https://user-images.githubusercontent.com/2349007/36911135-3f1482ce-1e08-11e8-8fc8-77e363e05dd6.png)

I think this is our baseline, and we can reference this issue in the future as we improve and learn what changes impact this load time.

**Comment by aappling-usgs** [comment]
#25 should help with this. so could presenting a low-resolution set of county boundaries for the nation view, only displaying (and loading) states and not counties for the nation view, and/or combining the state and nation topo lines into a single file.

**Comment by jread-usgs** [comment]
Just so we know what we are shooting for before release - an average time before users see a map of < 400ms. (Jim + Jordan's opinion)

**Comment by jread-usgs** [comment]
Seems google doesn't think we are gzipping the content: 
![image](https://user-images.githubusercontent.com/2349007/36914701-d27ff0a6-1e13-11e8-8ba8-fcf4e093c32c.png) (from a chrome devtools `audit`)

**Comment by aappling-usgs** [comment]
Yes, David got it :+1: - compression should exist on beta and prod

**Comment by aappling-usgs** [comment]
this issue is also being addressed in #97, #111, probably some others
